.travis.yml

```diff
@@ -19,14 +15,18 @@ node_js:
     - '5.0'
 
 env:
-    - INFLUX_VERSION=0.9
-    - INFLUX_VERSION=0.10
+    - INFLUX_VERSION=0.9 JUTTLE_PACKAGE=juttle/juttle
+    - INFLUX_VERSION=0.9 JUTTLE_PACKAGE=juttle@0.4.x
+    - INFLUX_VERSION=0.10 JUTTLE_PACKAGE=juttle/juttle
+    - INFLUX_VERSION=0.10 JUTTLE_PACKAGE=juttle@0.4.x
 
 services:
     - docker
 
 before_script:
-    - npm install juttle@0.4.x
+    - rm -rf node_modules/juttle
+    - npm install $JUTTLE_PACKAGE
```

Comment:
Should there even be a `$JUTTLE_PACKAGE` env variable to begin with ? the master line of any adapter should just work against the top of the juttle master ? That's why I'm asking @demmer since I'm not sure were we left this but it was either going to be `npm install juttle` or `npm install juttle/juttle`